source/languages/en/riakcs/cookbooks/configuration/Configuring-Riak-CS.md

```diff
@@ -26,39 +27,45 @@ that file. That section looks something like this:
 ]},
 ```
 
-The sections below walk you through some of the main configuration
-categories that you will likely encounter while operating Riak CS. For a
-comprehensive listing of available parameters, see the [[Full
-Configuration Listing|Configuring Riak CS#Full-Configuring-Listing]]
-section below.
+If you're updating from a version of Riak CS prior to 2.0.0 -- when the
+`riak-cs.conf` file was introduced -- you can still use the old-style
+`app.config` configuration file in place of the `riak-cs.conf`/`advanced.config`
+pair. The `app.config` file has identical syntax to the `advanced.config` file,
+so any examples that use the `advanced.config` syntax can be directly translated
+to the `app.config` file.  
+Please note that the older`app.config` file supersedes the newer configuration
```

Comment:
Line 35 is actually `to the \`app.config\` file.  `, and those two trailing spaces translate to a `</br>`. I went ahead and changed it to a newline (and thereby a `<p>`) though, to make the file more readable in non-markdown aware editors.